MicrosoftFluentUI.podspec

```diff
@@ -284,7 +285,8 @@ Pod::Spec.new do |s|
   s.subspec 'DatePicker_mac' do |datepicker_mac|
     datepicker_mac.platform = :osx
     datepicker_mac.dependency 'MicrosoftFluentUI/Core_mac'
-    datepicker_mac.source_files = ["macos/FluentUI/DatePicker/**/*.{swift,h}"]
+    datepicker_mac.source_files = ["macos/FluentUI/DatePicker/**/*.{swift,h}",
+                                    "macos/FluentUI/Extensions/Apperance+Theme.swift"]
```

Comment:
We don't use ColorSet and DynamicColor in the DatePicker, right?

We can create indivitual folders and subspecs for AppearanceExtensions.swift and DynamicColor.swift.

That way DatePicker can depend only on the AppearanceExtensions.

Reply:
Yes, that's correct. When you refer to individual extension, I assume you are referring to folders within extension folder? 
And have sub-spec as the same structure?
```
Extension 
 - AppearanceExtensions
 - DynamicColor
 ```

Reply:
just added another iteration based on your feedback.

Reply:
Just updated iteration with different subspecs and it complains because DynamicColor does use Appearance to determine the theme information.